fabric/fabric-commands/src/main/java/io/fabric8/commands/ProfileEditAction.java

```diff
@@ -70,6 +71,7 @@
     static final String EXT_PREFIX = "ext.";
     static final String DELIMITER = ",";
     static final String PID_KEY_SEPARATOR = "/";
+    static final String PROFILE_EDIT_INPUT_REGEX="^..*/..*=.*";
```

Comment:
Why do you use `..*` here?  Why cannot `.+` simply be used?

Reply:
Done. Replaced with PROFILE_EDIT_INPUT_REGEX="^.+/.+=.*";